content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -106,10 +108,12 @@ each submission:
 
 #. Log into the company for which the submission has to be done.
 #. Go to :guilabel:`General Settings`, and in the :guilabel:`Users` section, click
-   :guilabel:`Manage Users`. Select the user who will submit the VAT return.
+   :guilabel:`Manage Users`. Select the user who is connected to HMRC.
 #. Go to the :guilabel:`UK HMRC Integration` tab and click :guilabel:`Reset Authentication
-   Credentials` or :guilabel:`Remove Authentication Credentials` button. Then click again to create
-   new credentials. It is now possible to submit the tax report for this company.
+   Credentials` or :guilabel:`Remove Authentication Credentials` button.
+#. You can now :ref:`register your company to HMRC
+   <content/applications/finance/fiscal_localizations/register-company-to-hmrc>` and submit the tax
```

Comment:
```suggestion
   <uk_localization/hmrc-registration>` and submit the tax
```

Reply:
Sorry for late response, I was on leave the past 2 weeks.

I've made the change you suggested :)